templates/cloudflare-workers/package.json

```diff
@@ -3,31 +3,30 @@
   "sideEffects": false,
   "scripts": {
     "build": "remix build",
-    "deploy": "wrangler publish",
-    "dev:remix": "remix watch",
-    "dev:miniflare": "cross-env NODE_ENV=development miniflare ./build/index.js --watch",
-    "dev": "remix build && run-p \"dev:*\"",
-    "start": "cross-env NODE_ENV=production miniflare ./build/index.js"
+    "dev": "npm run build && concurrently \"npm:dev:remix\" \"npm:dev:cloudflare\"",
+    "dev:remix": "cross-env NODE_ENV=development remix watch",
+    "dev:cloudflare": "cross-env NODE_ENV=development wrangler dev --local --config wrangler.dev.toml",
+    "start": "npm run dev:cloudflare"
   },
   "dependencies": {
+    "@cloudflare/kv-asset-handler": "^0.2.0",
     "@remix-run/cloudflare": "*",
-    "@remix-run/cloudflare-workers": "*",
```

Comment:
Is it intentional that we're not downloading `@remix-run/cloudflare-workers` by default anymore?